generic_chooser/views.py

```diff
@@ -21,6 +21,11 @@
 from wagtail.search.backends import get_search_backend
 from wagtail.search.index import class_is_indexed
 
+if django.VERSION >= (3, 1):
+    from django.urls import re_path
+else:
+    from django.conf.urls import url as re_path
```

Comment:
You should be able to import directly from `django.urls` since the beginning - https://github.com/django/django/blob/stable/2.0.x/django/urls/__init__.py

Reply:
Good call. I did not even check that.